GTFS.DB.PostgreSQL/Collections/PostgreSQLShapesCollection.cs

```diff
@@ -190,6 +190,35 @@ public bool Remove(string entityId)
             }
         }
 
+        /// <summary>
+        /// Removes a range of entities by their IDs
+        /// </summary>
+        /// <param name="entityIds"></param>
+        /// <returns></returns>
+        public void RemoveRange(IEnumerable<string> entityIds)
+        {
```

Comment:
This should work, I don't know why Sqlite transactions are implemented differently. It would be interesting to try swapping the statements in the same way i mentioned above and see if it affects performance. Maybe a test for another day though